features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -642,181 +656,205 @@ class InfoBar extends React.Component {
-                                                                            text={epUrl}
-                                                                            onCopy={() => this.onCopy('prodUrlCopied')}
+                                                                            <CopyToClipboard
+                                                                                text={epUrl}
+                                                                                onCopy={() => this.onCopy('prodUrlCopied')}
+                                                                            >
+                                                                                <FileCopy color='secondary' />
+                                                                            </CopyToClipboard>
+                                                                        </Tooltip>
+                                                                    </Grid>
+                                                                )}
+                                                                {endpoint.URLs.https !== null && (
+                                                                    <Grid item xs={12}>
+                                                                        <TextField
+                                                                            defaultValue={endpoint.URLs.https}
+                                                                            id='bootstrap-input'
+                                                                            InputProps={{
+                                                                                disableUnderline: true,
+                                                                                classes: {
+                                                                                    root: classes.bootstrapRoot,
+                                                                                    input: classes.bootstrapInput,
+                                                                                },
+                                                                            }}
+                                                                            InputLabelProps={{
+                                                                                shrink: true,
+                                                                                className: classes.bootstrapFormLabel,
+                                                                            }}
+                                                                        />
+                                                                        <Tooltip
+                                                                            title={prodUrlCopied
+                                                                                ? 'Copied' : 'Copy to clipboard'}
```

Comment:
localize

Reply:
This has been modified with https://github.com/wso2/carbon-apimgt/commit/8e15048e7e670bb1965e778c0220c89331420d49